### DISCUSSION TITLE
If Bluetooth is not selected, disable use of the bluetooth stack

### DISCUSSION THREAD

**Discussion by lasantiagoa** [danleetw/FT8TW#6] [Ideas]
I usually use Bluetooth headphones with my phone. When using FT8CW connected via USB to my FX-4CR, the app still tries to connect to my headphones, and the sound card changes to my headphones. 

If we're using a USB or network, Bluetooth should not be used at all, and the app shouldn't try to connect or use the headphones or even activate the Bluetooth stack.

73 de Luis, HB9HJU

**Comment by danleetw** [comment]
> I usually use Bluetooth headphones with my phone. When using FT8CW connected via USB to my FX-4CR, the app still tries to connect to my headphones, and the sound card changes to my headphones.
> 
> If we're using a USB or network, Bluetooth should not be used at all, and the app shouldn't try to connect or use the headphones or even activate the Bluetooth stack.
> 
> 73 de Luis, HB9HJU

Fix，Try
https://github.com/danleetw/FT8TW/releases/tag/v0.924e
